coursebuilder/modules/explorer/_static/components/pages/homepage.html

```diff
@@ -182,47 +156,63 @@ <h2 class="section-label">Testimonials</h2>
         <div>
           <div class="card-tabs">
             <div class="row">
-              <div class="testimonials-card-tabs">
-                <ul class="tabs tabs-fixed-width">
-                  <li class="tab"><a class="active" href="#testimonial1">adrian</a></li>
-                  <li class="tab"><a href="#testimonial2">rachal</a></li>
-                  <li class="tab"><a href="#testimonial3">mike</a></li>
-                </ul>
-              </div>
+            <div class="testimonials-card-tabs">
+            <ul class="tabs tabs-fixed-width">
+              <li class="tab"><a class="active" href="#testimonial1">STUDENT: NAOMI</a></li>
+              <li class="tab"><a href="#testimonial2">TRAINER: NKECHI /FADWA</a></li>
+              <li class="tab"><a href="#testimonial3">JOURNO: NANCY AGUTU (FROM THE STAR)</a></li>
+              <li class="tab"><a href="#testimonial4">UNIVERSITY: TR.WAMBUI</a></li>
+              <li class="tab"><a href="#testimonial5">NEWSROOM: NAN/ AWOKO/STAR NEWSPAPER</a></li>
+            </ul>
+          </div>
             </div>
           </div>
           <div class="card-content grey lighten-4">
             <div id="testimonial1" class="testimonial-div">
               <div class="testimonial-card">
                 <h3>Words to live by</h3>
-                <p>Adrian Simpson set up a basic website for his new speechwriting business and
-                  began
-                  advertising online through search.
+                <p>“StoryLab Academy has provided an avenue to learn courses not offered in my school. Given that I am a writer, courses like verification will enable me write better stories.”
                   <br>
-                  <br>Within 31 days, he'd sold 31 speeches.</p>
+                  <br>
+                “As a journalism student, the Online course supplements what is taught in class and gives me a leg up. I am able to keep up with industry standards and be ready for employment.”
+                  </p>
               </div>
               <img src="/modules/explorer/_static/images/testimonial1.jpg" alt="testimonial image">
             </div>
             <div id="testimonial2" class="testimonial-div">
               <div class="testimonial-card">
                 <h3>Words to live by</h3>
-                <p>Adrian Simpson set up a basic website for his new speechwriting business and
-                  began
-                  advertising online through search.
+                <p>Adrian Simpson set up a basic website for his new speechwriting business and began advertising online through search.
```

Comment:
@Sharonsyra is this the right content?

Reply:
Muthoni added new data that i was supposed to update. It is in the word document.